.github/workflows/ci.yml

```diff
@@ -29,9 +29,9 @@ jobs:
           - name: "Lint: flake8"
             python: "3.11"
             tox: flake8
-          - name: "Lint: mypy"
+          - name: "Lint: pyright"
```

Comment:
Good idea, I did not know upright....but I just tested on a couple of other projects, one being a factor bigger than mopidy, and upright was a lot faster.

so 👍